extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -87,6 +83,10 @@ export function safeframeListener(event) {
     dev().warn(TAG, `Safeframe Host for sentinel: ${sentinel} not found.`);
     return;
   }
+  if (!safeframeHost.equalsSafeframeContentWindow(event.source)) {
```

Comment:
I think this might be a bug, `this.iframe_` is not set on the first message until `connectMessagingChannel` is called a few lines below.

Reply:
Caleb is right, this check fails in development mode. This is going to have to be a bigger PR than we originally thought it would be. 

![image](https://user-images.githubusercontent.com/5777995/76009634-875da280-5edf-11ea-9d0b-3cf84c01ff6f.png)

Reply:
Actually, nevermind. Changed the check to this and everything seems to be working. 
``` 
return otherWindow === this.baseInstance_.iframe.contentWindow
```